ports/openmvg/portfile.cmake

```
@@ -83,7 +71,7 @@ if(NOT VCPKG_BUILD_TYPE)
   file(RENAME "${CURRENT_PACKAGES_DIR}/debug/lib/openMVG/cmake" "${CURRENT_PACKAGES_DIR}/debug/share/openMVG/cmake")
 endif()
 
-vcpkg_cmake_config_fixup()
+vcpkg_cmake_config_fixup(PACKAGE_NAME openMVG)
```

```suggestion
vcpkg_cmake_config_fixup()
```
This option is a misnomer. It determines the target directory name. CMake accepts lower case, and the default is the port name, and `share/openmvg` already exists.

---

This is a fix for an installation [issue](https://github.com/microsoft/vcpkg/issues/43106) on Linux. Many other ports used a custom folder name to install CMake configuration files. I don't think that patching library's CMake scripts is better solution.

---

The issue needs separate investigation. This is covered in in CI. Please restore the original line. It is not a "fmt" topic.

---

> I changed the PR title. Now all fixes are in the "topic".

I'm not sure that this is the path to a quick merge of fmt and eigen updates.